plugins/fs-watch/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tauri-plugin-fs-watch-api",
+  "name": "@tauri-apps/plugin-fs-watch",
```

Comment:
could we just remove `fs-wtach` and `fs-extra` plugins and merge them to `fs` plugin instead?

Reply:
`fs-extra` was removed already, but there's some documentation to fix.

Reply:
and yeah `fs-watch` should be on `fs` under a feature flag. We can do that later.

Reply:
definitely before release, seems redundant to release it now and remove it later.

Reply:
I'll work on it.